dspace-server-webapp/src/main/java/org/dspace/app/rest/security/WebSecurityConfiguration.java

```diff
@@ -141,7 +142,8 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 // On logout, clear the "session" salt
                 .addLogoutHandler(customLogoutHandler)
                 // Configure the logout entry point & require POST
-                .logoutRequestMatcher(new AntPathRequestMatcher("/api/authn/logout", HttpMethod.POST.name()))
+                .logoutRequestMatcher(PathPatternRequestMatcher.withDefaults().matcher(HttpMethod.POST,
```

Comment:
Why not use the shorter variant `.logoutUrl("/api/authn/logout")` ?

If CSRF protection is enabled (default in DSpace REST), then the request must also be a POST. This means that by default POST "/logout" is required to trigger a log out.